package.json

```diff
@@ -18,41 +18,42 @@
   "devDependencies": {
     "babel-eslint": "^8.1.2",
     "babel-jest": "^22.0.4",
-    "babel-loader": "^7.1.2",
+    "babel-loader": "^7.1.4",
     "babel-plugin-transform-class-properties": "^6.23.0",
     "babel-plugin-transform-runtime": "^6.15.0",
     "babel-preset-es2015": "^6.18.0",
     "babel-preset-flow": "^6.23.0",
     "babel-register": "^6.26.0",
     "benchmark": "^2.1.4",
     "check-dependencies": "^1.1.0",
-    "css-loader": "^0.28.8",
+    "css-loader": "^0.28.11",
     "eslint": "^4.14.0",
     "eslint-plugin-flowtype": "^2.40.1",
-    "extract-text-webpack-plugin": "^3.0.2",
```

Comment:
`extract-text-webpack-plugin` doesn't support webpack 4 yet, `mini-css-extract-plugin` is used instead: https://github.com/webpack-contrib/extract-text-webpack-plugin/issues/701